server/py/services/api/crud/model_monitoring/deployment.py

```diff
@@ -1380,7 +1379,6 @@ async def create_model_endpoints(
             tuple[
                 mlrun.common.schemas.ModelEndpoint,
                 mm_constants.ModelEndpointCreationStrategy,
-                str,
```

Comment:
please also fix the desc of the func about the mep instructions that now doesn't include the model path.